src/test/kotlin/com/viartemev/thewhiterabbit/publisher/ConfirmPublisherTest.kt

```diff
@@ -15,40 +16,42 @@ import kotlinx.coroutines.delay
 import kotlinx.coroutines.runBlocking
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.fail
+import org.mockito.AdditionalMatchers
 import java.io.IOException
 import java.util.concurrent.atomic.AtomicInteger
 
 
 class ConfirmPublisherTest {
+    private val poisonMessage = OutboundMessage("", "", MessageProperties.PERSISTENT_BASIC, "poisonMessage")
+    private val validMessage = OutboundMessage("", "", MessageProperties.PERSISTENT_BASIC, "validMessage")
 
     @Test
-    fun test_1() {
-        val channel = mock<Channel> {
-            on { basicPublish(any(), any(), any(), any()) } doThrow IOException("Boom")
-        }
+    fun `publish several messages with one poison message`() {
+        val channel = mock<Channel>()
+        doNothing().`when`(channel).basicPublish(any(), any(), any(), AdditionalMatchers.aryEq("validMessage".toByteArray()))
+        doThrow(IOException("Boom")).`when`(channel).basicPublish(any(), any(), any(), AdditionalMatchers.aryEq("poisonMessage".toByteArray()))
+
         val counter = AtomicInteger()
         val confirmPublisher = ConfirmPublisher(channel)
```

Comment:
Here and at some other lines IDEA can't help complaining constructor/members are internal. Although gradle builds and runs tests nicely. What am I missing?

/home/wrover/src/the-white-rabbit/src/test/kotlin/com/viartemev/thewhiterabbit/publisher/ConfirmPublisherTest.kt
Error:(35, 32) Kotlin: Cannot access '<init>': it is internal in 'ConfirmPublisher'
Error:(66, 37) Kotlin: Cannot access 'continuations': it is internal in 'ConfirmPublisher'
Error:(76, 32) Kotlin: Cannot access '<init>': it is internal in 'ConfirmPublisher'
Error:(89, 37) Kotlin: Cannot access 'continuations': it is internal in 'ConfirmPublisher'